### PR TITLE
chore: Remove unused component and icons

### PR DESCRIPTION
**Pull request by yanguoyu**
1. Remove unused components. `CustomizedAsset `, `SettingTabs`, `Dropdown`, `experimentalRibbon`.
2. Remove unused icons and merge the same icons, add icon story to find icons easily.
3. Remove useless logic because the setting page is not in a new window.

I suggest that we'd better export and import the icon components in `packages/neuron-ui/src/widgets/Icons/icon.tsx`.
Then we can view the icons when we need to use or add them.

image

We need to check these icons and ensure the basic feature is running success will be ok.
image

image

image

image

image

This icon only shows when the language is English.
image

image

image

image

image

image

image

image

image

### CODE REVIEW

File: packages/neuron-ui/src/components/DataSetting/dataSetting.module.scss [hunk 1]
```diff
@@ -191,13 +196,26 @@
 .footer {
   padding: 24px 0;
   @include dialog-footer;
+  --fotter-background-color: #ecf9f0;
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/DataSetting/dataSetting.module.scss:199]: `fotter` should be `footer`

File: packages/neuron-ui/src/locales/en.json [hunk 2]
```diff
@@ -97,7 +97,7 @@
       },
       "actions": {
         "close": "Close",
-        "rescan": "Reconnect",
+        "reconnect": "Reconnect",
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/en.json:100]: The `key` `rescan` is not changed in codebase

Ref:
 `hardware-sign.actions.rescan`: https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/components/HardwareSign/index.tsx#L438

Comment (yanguoyu) [reply packages/neuron-ui/src/locales/en.json:100]: ` https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/components/HardwareSign/index.tsx#L438` is `hardware-sign.actions.rescan`, but not `hardware-verify-address.actions.rescan`

Comment (Keith-CY) [reply packages/neuron-ui/src/locales/en.json:100]: I see, `hardware-verify-address.actions.reconnect` is used at https://github.com/nervosnetwork/neuron/blob/c60d9367cf2c7640df06799d2dfb2eb367a98ee4/packages/neuron-ui/src/components/VerifyHardwareAddress/index.tsx#L210

**Review by devchenyan — APPROVED** [review]

**Comment by yanguoyu** [comment]
@silySuper Please help me to test this PR.

**Comment by silySuper** [comment]
/package.

**Comment by silySuper** [comment]
1. All UI above has been checked.
2. Main flow  has been checked.Offine sign status sometimes shows failed or pop up alert like this API Controller:	channel handling error: Error: {"code":-301,"message":"TransactionFailedToResolve: Resolve failed Unknown(OutPoint(0xc941c903d6b377f34ec3e1bf98b94952f7edf617f774cab8c0edb5fea4d3005800000000)). 
截屏2023-09-07 10 17 18

Log:

[neuron_debug_1694052856001.zip](https://github.com/nervosnetwork/neuron/files/12543952/neuron_debug_1694052856001.zip)

My Sysn status is 100%,what is the reason?

**Comment by yanguoyu** [comment]
> 1. All UI above has been checked.
> 2. Main flow  has been checked.Offine sign status sometimes shows failed or pop up alert like this API Controller:	channel handling error: Error: {"code":-301,"message":"TransactionFailedToResolve: Resolve failed Unknown(OutPoint(0xc941c903d6b377f34ec3e1bf98b94952f7edf617f774cab8c0edb5fea4d3005800000000)).
> 
> 截屏2023-09-07 10 21 35
> Log:
> 
> [neuron_debug_1694052856001.zip](https://github.com/nervosnetwork/neuron/files/12543952/neuron_debug_1694052856001.zip)
> 
> My Sync status is 100%,what is the reason?

The outpoint cell has spent. https://pudge.explorer.nervos.org/transaction/0x9cfa455415f8aca75ea25f84b68f2101523a899124a0a1fcc300d5ec0082d4de

**Comment by silySuper** [comment]
OK，after use get_live_cell method ，the tx_hash  cell status is unknown.

**Review by silySuper — APPROVED** [review]

**Pull request merged**